packages/bap-veri/bap-veri.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+name: "bil-veri"
+version: "0.1"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bil-verification/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bil-verification/issues"
+dev-repo: "https://github.com/BinaryAnalysisPlatform/bil-verification/"
+license: "MIT"
+
+build: [
+  ["oasis" "setup"]
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+
+install: [[make "install"]]
+
+remove: [["ocamlfind" "remove" "bil-veri"]]
```

Comment:
it should be `bap-veri`. Also, you forgot to remove the binary "bil-ver" (btw, we acknowledged, that it should be `bap-veri`.